doc/lightning-plugin.7.txt

```diff
@@ -0,0 +1,45 @@
+LIGHTNING-PLUGIN(7)
+===================
+:doctype: manpage
+
+NAME
+----
+lightning-plugin - Manage plugins with RPC
+
+SYNOPSIS
+--------
+*plugin* command [parameter] [second_parameter]
+
+DESCRIPTION
+-----------
+
+The *plugin* RPC command allows to manage plugins without having to restart lightningd.
+It takes 1 to 3 parameters: a command (start/stop/adddir/reload/list) which describes the
+action to take and optionally one or two parameters which describes the plugin on which
+the action has to be taken.
+
+The 'start' command takes a path as parameter and will load the plugin available from this
```

Comment:
The help description contains example, but manpage does not?

Reply:
I made a `cli command parameter` in verbose help to make it concise. I can replicate it in the manpage too, but it seems already more detailed.

Reply:
Generally manpage is more detailed and includes things like possible errors.  Maybe okay for now.